cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -508,6 +510,10 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
           resultTable()
         }
 
+        note {
+          p("Note that no specific privileges are required for showing the current user's privileges using `SHOW USER [username] PRIVILEGES`.")
```

Comment:
In line 501 we use `SHOW USER username PRIVILEGES`, and in this line we use `SHOW USER [username] PRIVILEGES` - it seems that the first one is correct?

Reply:
`SHOW USER username PRIVILEGES`is the usual command but in 4.1 you can also type `SHOW USER PRIVILEGES` without any username to get your own privileges. What I tried to write here is that you can run `SHOW USER PRIVILEGES` as well as `SHOW USER username PRIVILEGES` with your own username without any specific privilege. 

If you find any clearer way of formulating that, please go ahead

Reply:
thanks @Lojjs - I pushed a small update to add that users can run `SHOW USER username PRIVILEGES` or `SHOW USER PRIVILEGES` for showing the current user's privileges

Reply:
Thanks looks good